Assets/Scripts/FollowingCamera.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FollowingCamera : MonoBehaviour
+{
+    public GameObject followingCamera;
+    Transform cameraTransform;
+    [Range(0.0f, 1.0f)]
+    public float followSpeed = 0.5f;
+
+    Transform myTransform;
+
+    public void SetFollowingCamera(GameObject obj)
+    {
+        
+        followingCamera = obj == null ? null : obj;
+        cameraTransform = obj == null ? null : obj.GetComponent<Transform>();
+    }
+
+    void Awake()
+    {
+        SetFollowingCamera(followingCamera);
+        myTransform = GetComponent<Transform>();
+    }
+
+    void Start()
+    {
+        if (followingCamera == null)
+        {
+            GameObject camera = GameObject.FindGameObjectWithTag("MainCamera");
```

Comment:
Not sure if this is for many cameras/players anymore if we get only `MainCamera`